module1.js

```diff
@@ -28,7 +32,11 @@ function question1(name1) {
 */
 
 function question2(number1, number2) {
-  // TODO:
+  if (number1 == number2) {
```

Comment:
Remember `==` includes type casts, so e.g. `'1' == 1`; it's generally better to use `===`.

Reply:
thanks

Reply:
I run out of time and I could only take a look on module 2.
Thank you and I'll try to pay more attention next time. Most of the time when I work fast, I miss.